library/src/main/java/com/schibsted/spain/barista/cleardata/ClearFilesRule.java

```diff
@@ -0,0 +1,33 @@
+package com.schibsted.spain.barista.cleardata;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+/**
+ * This rule clears all app's stored files before running each test
+ */
+public class ClearFilesRule implements TestRule {
+
+  @Override
+  public Statement apply(final Statement base, Description description) {
+    return new Statement() {
+      @Override
+      public void evaluate() throws Throwable {
+        clearData();
+        base.evaluate();
+        clearData();
+      }
+    };
+  }
+
+  private void clearData() {
+    FileOperations.deleteAllFiles(getAppContext());
```

Comment:
Oh, I'm working on a screen that saved things at the cache dir: `obj_is = new FileInputStream(new File(c.getCacheDir().getPath(), filename));`

Well, let's see how this Rule works in the real world. If we need to extend it, we will do it for sure :·)